src/commands/emojis/emojiinfo.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from 'yuuko'
+import { MessageEmbed } from 'helperis'
+import GamerClient from '../../lib/structures/GamerClient'
+
+export default new Command([`emojiinfo`, `emoji`], async (message, args, context) => {
+  if (!message.guildID || !message.member) return
+
+  const Gamer = context.client as GamerClient
+  const language = Gamer.getLanguage(message.guildID)
+
+  const [nameOrID] = args
+  if (!nameOrID) return message.channel.createMessage(language(`emojis/emojiinfo:NEED_NAME_OR_ID`))
+
+  // Find the emoji in the guild
+  const emoji = message.member.guild.emojis.find(emoji =>
+    parseInt(nameOrID) ? emoji.id === nameOrID : emoji.name === nameOrID
+  )
+
+  if (!emoji) return message.channel.createMessage(language(`emojis/emojiinfo:DOESNT_EXIST`, { nameOrID }))
+
+  const embed = new MessageEmbed()
+    .setAuthor(message.member.guild.name, message.member.guild.dynamicIconURL(undefined, 2048))
```

Comment:
```suggestion
    .setAuthor(message.member.guild.name, message.member.guild.iconURL)
```

author icons are small enough we dont need to make it fetch a huge 2048 size image